source/infrastructure/pipeline/pipeline-stack.ts

```diff
@@ -51,6 +52,28 @@ class PipelineStack extends Stack {
     pipeline.addStage(deployStage, {
       post: [this.getIntegrationTestStep({})],
     });
+
+
+
+    //pipeline must be built before findings can be suppressed
+    pipeline.buildPipeline();
+    NagSuppressions.addStackSuppressions(this, [
+      {
+        id: "AwsSolutions-IAM5",
+        reason: "necessary permissions for being able to self-mutate"
+      },
+      {
+        id: "AwsSolutions-CB4",
+        reason: "Update step provided by construct"
+      }
+    ],true)
```

Comment:
I would rather add these to the resources that require them than add to all children, even if it's in the pipeline. Are there a lot of findings that need to be suppressed? Or are they difficult to suppress because they change when the project changes? If that's the case we should probably add these as stack-level suppressions.

Also please configure your editor to insert semicolons.

Reply:
I'll take a look and try to remove the child flags. I think I just copy-pasted it to all suppressions because it was the only way I found to get one of the earlier suppressions to work and didn't really think beyond it (have fixed that first suppression now)